metecho/api/push.py

```diff
@@ -54,7 +54,10 @@ async def push_message_about_instance(instance, message):
     new_message["model_name"] = model_name
     new_message["id"] = id
     sent_message = {"type": "notify", "content": new_message}
-    if await get_set_message_semaphore(channel_layer, sent_message):
+    not_deleted = getattr(instance, "deleted_at", None) is None
+    message_about_delete = "DELETE" in message["type"]
```

Comment:
I think this should include `REMOVE` also, so it allows the `SCRATCH_ORG_REMOVE` event.